dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/template/impl/DefaultHTMLTemplate.java

```diff
@@ -116,9 +116,9 @@ private String getTextTypeMessage(String content,boolean showAll){
             }
 
             StringBuilder contents = new StringBuilder(100);
-            for (String str : list){
+            for (Object obj : list){
```

Comment:
the type of 'list' variable is List<String> , why replace String with Object

Reply:
JSONUtils.toList will return List< LinkedHashMap > and will Cause exception : LinkedHashMap cannot be cast to String